slices/secureboot-db.yaml

```diff
@@ -0,0 +1,19 @@
+package: secureboot-db
+
+essential:
+  - secureboot-db_copyright
+
+slices:
+  dist:
+    essential:
+      - sbsigntool_bins
```

Comment:
afaik this is a binary data file, so why would it need this dependency?

Reply:
In theory it doesn't have to - but the secureboot-db package pulls in sbsigntools, and it was to keep this dependency

Reply:
yeah but I think it's actually the `services` slice that needs that, cause the service will do:

```
ExecStart=/usr/bin/sbkeysync --no-default-keystores --keystore /usr/share/secureboot/updates --verbose
```

Reply:
Right good point - i'll move the dep to that slice